vllm/model_executor/custom_op.py

```diff
@@ -138,15 +156,29 @@ def default_on() -> bool:
     # - MyOp.enabled()
     # - op_registry["my_op"].enabled()
     op_registry: dict[str, type['CustomOp']] = {}
+    op_registry_oot: dict[str, type['CustomOp']] = {}
 
     # Decorator to register custom ops.
+    # For OOT custom ops:
```

Comment:
Do you have some context behind how you wanted to use this (for HPU?). For example, is it going to ship a separate package with the custom operators?

Reply:
How do you plan to deal with oot collisions? If someone installs two packages (one for HPU, one for another accelerator), what wins out?